### PR TITLE
Update template [README.md and template.keyboard_info]

### PR DESCRIPTION
**Pull request by MakaraSok**
- Deployed section in README.md has been removed.
- template.keyboard_info has been added to release/template

### CODE REVIEW

**Comment by DavidLRowe** [comment]
I'd suggest some additional changes to README.md:
(1) Add "macOS" after "Windows" in the "Supported Platforms" section.
(2) Change `__DESCRIPTION__` to:

```
Description
-----------
<add a description of this keyboard>
```

(3) Change the Links to be more informative:

```
 * Home:     <___URL of keyboard info on your website___> or else omit this line
 * Help:     <___URL of keyboard help on your website___> or else omit this line
 * Contact:  <___EMAIL___> if you want to include your email address or else omit this line
```

(4) Change the "Todo" section to be more informative:

```
* Fill in the blanks in this README.md file:
  * Fill in the name of keyboard
  * Complete the copyright information: YYYY should be year (or range of years from date of first release to date of present release), followed by copyright holder (person or legally recognised entity)
  * Fill in the version number: change "A.B" to the version number of the present release; use "1.0" if this is a new keyboard
  * Fill in a description of the keyboard
  * Fill in the Links information (or delete the line if it doesn't apply)
  * Delete any platforms under the "Supported Platforms" section that your keyboard files do not support. The result should agree with the "store(&TARGETS)" information in the .kmn file. (If "store(&TARGETS)" is "any", then leave all the platforms in the list.)
* Update the LISCENSE.md file with the same copyright information used in the README.md file [note, to be included in the Keyman keyboards repo, you must use MIT license]
* Update HISTORY.md to give date of first release (and any other history you want)
```

**Comment by DavidLRowe** [comment]
I'd also suggest dropping the word "Template" from the beginning of the HISTORY.md file, or else changing it to something that will make it (more) obvious that it needs to be changed to the name of the keyboard. Many people just left the word "Template" there.

I think that there needs to be a lot more direction on how to fill out the .keyboard_info file. Perhaps something like [[put language name here]] where the language should go. 

There also needs to be instructions in the "Todo" section on renaming the template.keyboard_info file to match their project, by replacing "template" with the name of their project.

Thanks for your work on this. If we can make things clearer at the start, it will cut down on the things people have to correct later, and save time getting new keyboards approved.

**Comment by MakaraSok** [comment]
@DavidLRowe , I have updated the README.md and HISTORY.md according to you comments. Thank you!

**Comment by LornaSIL** [comment]
Some typos in the README.md file:
LISCENSE.md > LICENSE.md
desecription > description

Otherwise, it looks good.

File: release/template/README.md [hunk 1]
```diff
@@ -26,7 +27,14 @@ Supported Platforms
...
- * Fill in the blanks in this file
- * Update HISTORY.md to give date of first release (and any other history you want)
+* Fill in the blanks in this README.md file:
+  * Fill in the name of keyboard
+  * Complete the copyright information: YYYY should be year (or range of years from date of first release to date of present release), followed by copyright holder (person or legally recognised entity)
+  * Fill in the version number: change "A.B" to the version number of the present release; use "1.0" if this is a new keyboard
+  * Fill in a description of the keyboard
+  * Fill in the Links information (or delete the line if it doesn't apply)
+  * Delete any platforms under the "Supported Platforms" section that your keyboard files do not support. The result should agree with the "store(&TARGETS)" information in the .kmn file. (If "store(&TARGETS)" is "any", then leave all the platforms in the list.)
+* Update the LISCENSE.md file with the same copyright information used in the README.md file [note, to be included in the Keyman keyboards repo, you must use MIT license]
```
Comment (mcdurdin) [inline release/template/README.md]: typo here with `LISCNESE`

File: release/template/README.md [hunk 2]
```diff
@@ -26,7 +27,14 @@ Supported Platforms
...
+* Update HISTORY.md to give date of first release (and any other history you want)
+* Update template.keyboard_info by filling in the following sections "language," "desecription," and "related" if applicable. For the details of the specification of .keyboard_info file, please visit: https://help.keyman.com/developer/cloud/keyboard_info/1.0/ .
```
Comment (mcdurdin) [inline release/template/README.md]: typo here `desecription`

**Comment by MakaraSok** [comment]
@mcdurdin @LornaSIL Done. Thank you for catching that.

**Pull request merged**